pages/basics/content/animation.js

```diff
@@ -1,6 +1,29 @@
 module.exports = {
   title: 'Animation',
   subtitle: '',
-  description: ``.replace(/~/g, '`'),
+  description: `
+    Try this in [your browser](https://codesandbox.io/s/31VMyP7XO)
```

Comment:
For codesansbox links, you can actually do [this](https://github.com/kentcdodds/glamorous-website/blob/75101c53e97f5676a9ac23d0ae9f9cc4169939a2/pages/integrations/content/pseudo.js#L14-L15) which will render like [this](https://rc.glamorous.rocks/integrations). I can go ahead and update it for you, but in the future you can do that instead :) Thanks!